src/services/findAllReferences.ts

```diff
@@ -1231,6 +1237,8 @@ namespace ts.FindAllReferences {
         function isValidReferencePosition(node: Node, searchSymbolName: string): boolean {
             // Compare the length so we filter out strict superstrings of the symbol we are looking for
             switch (node.kind) {
+                case SyntaxKind.PrivateIdentifier:

```

Comment:
I think this could be 
```
case SyntaxKind.PrivateIdentifier:
case SyntaxKind.Identifier:
      return (node as Identifier | PrivateIdentifier).text.length === searchSymbolName.length;
```